test/system/065-cp.bats

```diff
@@ -99,18 +94,19 @@ load helpers
 
     # Test extract on non archive.
     run_podman cp --extract=true $srcdir/$rand_filename cpcontainer:/foo.txt
+    run_podman exec cpcontainer cat /foo.txt
+    is "$output" "$rand_content"
 
-    # Cannot extract an archive to a file!
-    run_podman 125 cp --extract=true $tar_file cpcontainer:/tmp/file.txt
-    is "$output" 'Error: cannot extract archive .* to file "/tmp/file.txt"'
+    # Extract an archive to a file.
+    run_podman cp --extract=true $tar_file cpcontainer:/tmp/file.txt
```

Comment:
This is a useless test:  `/tmp/$dirname` has already been created (lines 91-92 above).

Worse, I find this command and comment confusing. What does it mean to "extract an archive to a file"? I would expect this to fail, and don't understand why it doesn't.

By adding the `exec` below I can see that podman seems to be stripping off `file.txt` and extracting into the parent directory. I think this is counterintuitive, and even dare say it is wrong behavior; but that's not my decision to make. If it is correct, or if it's impossible to fix, I think the test should be amended in some way (either with the `exec rm` or, preferably, with an extraction into a different directory) and, also, the comment should be expanded to clarify this weird behavior.

```diff
diff --git a/test/system/065-cp.bats b/test/system/065-cp.bats
index b1c3fbf8a..db9fbfff6 100644
--- a/test/system/065-cp.bats
+++ b/test/system/065-cp.bats
@@ -98,6 +98,7 @@ load helpers
     is "$output" "$rand_content"

     # Extract an archive to a file.
+    run_podman exec cpcontainer rm -rf /tmp/$dirname
     run_podman cp --extract=true $tar_file cpcontainer:/tmp/file.txt
     run_podman exec cpcontainer cat /tmp/$dirname/$rand_filename
     is "$output" "$rand_content"
```